### PR TITLE
Add GraKel

### PR DESCRIPTION
**Pull request by jan-janssen**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by jan-janssen** [comment]
Opened a pull request upstream to include the license in the pip package https://github.com/ysig/GraKeL/pull/54

**Comment by jan-janssen** [comment]
@conda-forge/staged-recipes, ready for review

File: recipes/grakel/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+    - grakel
+    - grakel.datasets
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://ysig.github.io/GraKeL/
+  summary: A scikit-learn compatible library for graph kernels
```
Comment (dopplershift) [inline recipes/grakel/meta.yaml:47]: Should be able to remove this duplicated `about` block

File: recipes/grakel/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+  host:
+    - pip
+    - numpy
+    - python
+    - cython
+  run:
+    - cython >=0.27.3
```
Comment (dopplershift) [inline recipes/grakel/meta.yaml:27]: `cython` is only needed for build-time, no need to have it under `run`.

File: recipes/grakel/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+    - future >=0.16.0
+    - joblib
+    - nose >=1.1.2
```
Comment (dopplershift) [inline recipes/grakel/meta.yaml:30]: `nose` is only needed to run the test suite, so I don't think you need this here?

File: recipes/grakel/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+  host:
+    - pip
+    - numpy
+    - python
+    - cython
```
Comment (dopplershift) [inline recipes/grakel/meta.yaml:25]: Should this be `>=0.27.3`?

File: recipes/grakel/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,71 @@
...
+  summary: A scikit-learn compatible library for graph kernels
+  dev_url: https://github.com/ysig/GraKeL/
+  license: BSD-1-Clause
+  license_file: PLEASE_ADD_LICENSE_FILE
+
+about:
+  home: https://ysig.github.io/GraKeL/
+  license: BSD-3-Clause
+  license_family: BSD
+  license_file: LICENSE
```
Comment (dopplershift) [inline recipes/grakel/meta.yaml:56]: Would be good to have a comment reminding to remove the local copy of `LICENSE` once the upstream issue is resolved.

**Review by dopplershift — CHANGES_REQUESTED** [review]
Just a few odds and ends to take care of.

File: recipes/grakel/meta.yaml (hunk 5 above)
Comment (jan-janssen) [reply recipes/grakel/meta.yaml]: I added a comment

File: recipes/grakel/meta.yaml (hunk 4 above)
Comment (jan-janssen) [reply recipes/grakel/meta.yaml]: I added the version constraint

File: recipes/grakel/meta.yaml (hunk 3 above)
Comment (jan-janssen) [reply recipes/grakel/meta.yaml]: I removed nose

File: recipes/grakel/meta.yaml (hunk 2 above)
Comment (jan-janssen) [reply recipes/grakel/meta.yaml:25]: I removed cython from the run section

File: recipes/grakel/meta.yaml (hunk 1 above)
Comment (jan-janssen) [reply recipes/grakel/meta.yaml]: I removed the secondary about block.

File: recipes/grakel/meta.yaml (hunk 2 above)
Comment (jan-janssen) [reply recipes/grakel/meta.yaml:27]: I had to add cython again otherwise pip check fails.

File: recipes/grakel/meta.yaml (hunk 3 above)
Comment (jan-janssen) [reply recipes/grakel/meta.yaml:30]: Just like cython I also had to add nose again otherwise pip check fails.

**Comment by jan-janssen** [comment]
@dopplershift Can you take another look?

File: recipes/grakel/meta.yaml (hunk 2 above)
Comment (dopplershift) [reply recipes/grakel/meta.yaml:27]: Ugh. It's in the upstream package metadata. Ok, fine.

**Review by dopplershift — APPROVED** [review]

**Pull request merged**